boskos/resources.yaml

```diff
@@ -35,7 +35,9 @@ resources:
         - clusters:
           - machinetype: n1-standard-32
             numnodes: 3
-            version: 1.16
+            version: 1.17
+            releasechannel:
+              channel: RAPID
```

Comment:
isn't it channel or version? how does both work?

Reply:
They are in different config setting struct.
- `version` is in `ServerConfig`, where `ServerConfig` also has a field called `channel`.
- `ReleaseChannel` field is in &container.Cluster, which is used in the gke.go create cluster func.
Not quite clear about their differences. Seems a kind of overlapping.